src/notebooks/debugger/helper.ts

```diff
@@ -15,7 +15,9 @@ export enum IpykernelCheckResult {
 }
 
 export async function isUsingIpykernel6OrLater(kernel: IKernel): Promise<IpykernelCheckResult> {
-    const code = 'import ipykernel\nprint(ipykernel.__version__)';
+    const code = `import builtins
+import ipykernel
+builtins.print(ipykernel.__version__)`;
```

Comment:
@roblourens You could completely get rid of print as well.
```pyhton
import ipykernel
ipykernel.__version__
```

that should be sufficient, as the Python kernel prints the value of the last expression in this case `ipykernel.__version__`, hence a print isn't really necessary

Reply:
though thats an optional change

Reply:
I tried but I'm actually kind of struggling to do that. I get the return result in a different shape and I can't convince TS that the indexing that I have to do is valid... I think I'll leave it this way